JourneysSquad/JourneysSquad/View/HistoryOfOperationView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// View with a navigation bar, a title and a toolbar item.
+struct HistoryOfOperationView: View {
+    var body: some View {
+        NavigationStack {
+            Text("Any other view can be placed here")
+                .navigationTitle("History of Operations")
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    Button("Add") {}
+                        .font(.system(size: fontBodySize))
+                        .fontWeight(.bold)
+                        .foregroundColor(.blue)
+                }
+        }
```

Comment:
nit:

I'd suggest to make it this way.
It will allow more clear refactoring (in the following PR you will just replace everything inside the `VStack` with the new View(s).

Does that make sense?
```suggestion
        NavigationStack {
            VStack {
                Text("Any other view can be placed here")
            }
            .navigationTitle("History of Operations")
            .navigationBarTitleDisplayMode(.inline)
            .toolbar {
                Button("Add") {}
                    .font(.system(size: fontBodySize))
                    .fontWeight(.bold)
                    .foregroundColor(.blue)
            }
        }
```

Reply:
Yes, thanks!
Fixed.